.github/pull_request_template.md

```diff
@@ -11,7 +11,7 @@ https://github.com/apache/druid/blob/master/dev/committer-instructions.md#pr-and
 
 ### Description
 
-<!-- Describe the goal of this PR, what problem are you fixing. If there is a corresponding issue (referenced above), it's not necessary to repeat the description here, however, you may choose to keep one summary sentence. -->
+Describe the goal of this PR, what problem are you fixing. If there is a corresponding issue (referenced above), it's not necessary to repeat the description here, however, you may choose to keep one summary sentence. Focus on "User Experience", describe how users will interact with the changes you're submitting. It's important to get the user experience right, because if we release a feature with poor UX, then it's hard to change later without breaking compatibility. We'd like the master branch to be releasable at all times, which means we should figure UX out before committing a patch.
```

Comment:
i think this and the other change to template instructions should be left as a comment `<!-- ... -->`, also it might be more appropriate in the 2nd section, `Describe your patch: what did you change in code? How did you fix the problem? ...`

I'm not sure we really need the last two sentences to justify why we are asking for things, this template is already pretty long which I'm afraid the more stuff we add to it, the less likely people are to actually read it